client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.test.js

```diff
@@ -48,6 +48,83 @@ describe('DataFilesToolbar', () => {
   });
 });
 
+describe('DataFilesToolbar', () => {
+  it('do not render unnecessary buttons in Community Data', () => {
```

Comment:
Not really required, but I believe we try to make these `it` functions more or less readable, such that it would make sense grammatically
```suggestion
  it('does not render unnecessary buttons in Community Data', () => {
```